src/analytics/amplitude/index.ts

```diff
@@ -1,15 +1,15 @@
 import { Identify, identify, init, track } from '@amplitude/analytics-browser'
```

Comment:
this will get rebased after https://github.com/Uniswap/interface/pull/4653 is merged